sdk/src/main/java/co/omise/android/models/PaymentMethod.kt

```diff
@@ -39,8 +39,19 @@ data class PaymentMethod(
                         installmentTerms = when (sourceType) {
                             is SourceType.Installment -> SourceType.Installment.availableTerms(sourceType)
                             else -> null
+                        },
+                        banks = when (sourceType) {
+                            is SourceType.Fpx -> mockFpxBanks()
+                            else -> null
                         }
                 )
+
+        private fun mockFpxBanks(): List<Bank>? {
```

Comment:
I don't think we need mock data in the production. Could you remove it?

Reply:
@nuxzero Actually the capability object that's created from Specify Payment Methods mode doesn't have information about FPX banks, so I think we have to simulate it somehow.

Reply:
@nuxzero Moved the sample banks out of the SDK. https://github.com/omise/omise-android/pull/192/commits/5a8e2c4b83d6a0c0c9b8faa58dc81fa5d25dfd8d